subprojects/dependency-management/src/test/groovy/org/gradle/internal/resource/transfer/ProgressLoggingExternalResourceAccessorTest.groovy

```diff
@@ -99,9 +99,9 @@ class ProgressLoggingExternalResourceAccessorTest extends Specification {
         then:
         1 * progressLoggerFactory.newOperation(_) >> progressLogger
         1 * progressLogger.started()
-        1 * progressLogger.progress('1 KB/4 KB downloaded')
-        1 * progressLogger.progress('3 KB/4 KB downloaded')
-        1 * progressLogger.progress('4 KB/4 KB downloaded')
+        1 * progressLogger.progress('1 KB/4 KB downloaded: location')
```

Comment:
I just tried this out locally and it seems the console is missing some context information. I.e. it shows things like:

```
> compileJava > 3 KB/4KB downloaded: foobar.jar
```

When really it should be more like

```
> compileJava > resolving configuration 'compileClasspath' > resolving artifact foobar.jar > 3KB / 4KB downloaded
```

The former is a workaround at best. Should we insert more intermediate progress operations or wait for @eriwen and the core team to completely move towards build operations?